### PR TITLE
Remove unnecessary "Mac" mentions

### PR DESCRIPTION
**Pull request by williamboman**

### CODE REVIEW

**Comment by dufferzafar** [comment]
:+1:

File: README.md [hunk 1]
```diff
@@ -54,14 +54,14 @@
 
 ### Finder
 
-- [TotalFinder](http://totalfinder.binaryage.com/) - A powerful alternative to finder.
-- [XtraFinder](https://www.trankynam.com/xtrafinder/) - Adds useful features to the mac finder.
+- [TotalFinder](http://totalfinder.binaryage.com/) - A powerful alternative to Finder.app.
+- [XtraFinder](https://www.trankynam.com/xtrafinder/) - Adds useful features to Finder.app.
```
Comment (iCHAIT) [inline README.md]: Just `Finder` or `Finder.app`?

Comment (williamboman) [inline README.md]: I'm 50/50, your call :smile:.

Comment (iCHAIT) [inline README.md]: Lets keep it `Finder` :blush:

Comment (williamboman) [inline README.md]: Done

**Comment by iCHAIT** [comment]
:+1:

**Pull request merged**